### ISSUE TITLE
The 'sortOutput' option is passed to tsc

### ISSUE THREAD

**Issue by stijnherreman** [ivogabe/gulp-typescript#355]
**Expected behavior:**
The unofficial 'sortOutput' option is only used by gulp-typescript and not passed to tsc.

**Actual behavior:**
The unofficial 'sortOutput' option is passed to tsc, resulting in a non-blocking compiler error:

> error TS5023: Unknown compiler option 'sortOuput'.

**Your gulpfile:**
```javascript
gulp.task("typescript", ["install-packages"], function () {
    return gulp
        .src(["static/ts/*.ts", "static/ts/**/*.ts"])
        .pipe(sourcemaps.init())
        .pipe(typescript({ sortOuput: true, target: "ES5" }))
        .pipe(concat("all.js"))
        .pipe(sourcemaps.write())
        .pipe(gulp.dest("./static/ts"))
})
```

**Comment by ivogabe** [comment]
Looks like you misspelled it as `sortOuput`, you missed the `t` before the `p`. So it is a useful error in this case 😄. I did a quick check and `sortOuput` doesn't show that error.

**Comment by stijnherreman** [comment]
Ah good catch, thanks!
